index.js

```diff
@@ -192,6 +192,10 @@ module.exports = class CurrencyConverter {
      }
      return singleInstance;
   }
+  
+  static getDayEpoch(unixTsInMillis){
+     return Math.floor((unixTsInMillis ? unixTsInMillis : new Date())/86400000);
```

Comment:
I think it would be better to work consistently with the moment.js lib. for example you can call moment.startOf('day') for both dates and then compare using isAfter, isSameOrAfter, isBefore, IsSameOrBefore etc.